Sources/ScipioKit/Executor.swift

```diff
@@ -45,7 +45,7 @@ struct ProcessExecutor: Executor {
             switch self {
             case .terminated(let result):
                 var errors = ["Execution was terminated:"]
-                if let output = try? result.unwrapStdErrOutput() {
+                if let output = try? result.unwrapOutput() {
```

Comment:
Currently, `xcbuild` and `xcodebuild` emits error log to standard output.
So it's comfortable to show standard outputs